gradle/installer-json.gradle

```diff
@@ -0,0 +1,115 @@
+import groovy.json.JsonOutput
+
+tasks.register("generateInstallerJson", GenerateInstallerJsonTask) {
+	configurations = [ common: "installer", development: "development"]
+	repositoryOverrides = [
+	    "mixinextras-fabric": "https://repo.maven.apache.org/maven2/"
+	]
```

Comment:
This needs to be duplicated into the launch wrapper json. Unless there is a specific reason why this wont work with LW?